BrowserKit/Sources/ComponentLibrary/ResizableButton.swift

```diff
@@ -18,12 +21,12 @@ class ResizableButton: UIButton {
         }
     }
 
-    override init(frame: CGRect) {
+    public override init(frame: CGRect) {
```

Comment:
Out of curiosity, is there a standard in the project regarding the use of 'public override' and 'override public'?

Reply:
I don't think we have a standard yet for this, worth discussing, I like have `public` first, but can sway another way haha

Reply:
I Like the opposite 😈

Reply:
I personally don't care - although I think that `override public var` is a clearer statement that `public override var` because you're overriding a public var vs publicly overriding a var. I've added it to this week's swiftlint's voting, so check that out in the slack. For now, let's leave it as is, and it'll get corrected whenever the rules goes in. :)

Reply:
Awesome there's a rule for this 💪 Thanks Roux